packages/libraries/cli/src/commands/schema/fetch.ts

```diff
@@ -39,6 +39,37 @@ const SchemaVersionForActionIdQuery = graphql(/* GraphQL */ `
   }
 `);
 
+const LatestSchemaVersionQuery = graphql(/* GraphQL */ `
+  query LatestSchemaVersion(
+    $includeSDL: Boolean!
+    $includeSupergraph: Boolean!
+    $includeSubgraphs: Boolean!
+  ) {
+    latestValidVersion {
```

Comment:
I tried to use a shared fragment fragment in `SchemaVersionForActionId` and `LatestSchemaVersionQuery` queries, but couldnt get the `graphql` parser to understand it.
I also considered creating a single document with multiple queries but `registryApi(...). request` doesn't support passing an operation name.
So I think a little duplication is the simplest.